docs/Introduction/about_us.md

```diff
@@ -2,11 +2,12 @@
 
 <div class="grid cards team" markdown>
 
--   ![Gianluca Colombo One Off tech Profile](https://avatars.githubusercontent.com/u/82368796?v=4){ width="400" }
+-   ![Gianluca Colombo OneOff-tech Profile](https://oneofftech.xyz/assets/images/team/gianluca.jpg){ width="400" }
```

Comment:
I had tried that initially, but for some reason the localisation plug-in struggled with caching an image from your website. Alternatively we can store and use a local version directly.